server/src/discovery.h

```diff
@@ -50,6 +50,7 @@ class Discovery
 
 private:
     void discover(const disco::Message& msg);
+    void addTask();
```

Comment:
why empty task?

Reply:
Just for single address. Need to be updated with full address thread pool.